src/Microsoft.DotNet.Compiler.Common/LibraryExporterExtensions.cs

```diff
@@ -27,7 +28,21 @@ public static void CopyTo(this IEnumerable<LibraryAsset> assets, string destinat
 
             foreach (var asset in assets)
             {
-                File.Copy(asset.ResolvedPath, Path.Combine(destinationPath, Path.GetFileName(asset.ResolvedPath)), overwrite: true);
+                var file = Path.Combine(destinationPath, Path.GetFileName(asset.ResolvedPath));
+                File.Copy(asset.ResolvedPath, file, overwrite: true);
+                RemoveFileAttribute(file, FileAttributes.ReadOnly);
+            }
+        }
+
+        private static void RemoveFileAttribute(String file, FileAttributes attribute)
```

Comment:
Can you move this method to be below the StructuredCopyTo method below. Since it is being used by it, it would ready nicer for it to be after it as well.